### DISCUSSION TITLE
v0.10.11

### DISCUSSION THREAD

**Discussion by AsamK** [AsamK/signal-cli#996] [Releases]
**Full Changelog**: https://github.com/AsamK/signal-cli/compare/v0.10.10...v0.10.11

<hr /><em>This discussion was created from the release <a href='https://github.com/AsamK/signal-cli/releases/tag/v0.10.11'>v0.10.11</a>.</em>

**Comment by moppman** [comment]
Lol, you implemented the "Stories" feature (the "receive" part, that is) before the official clients even get that functionality. Impressive!
Did you "reverse-engineer" from the Stories commits in the [official repo](https://github.com/signalapp/Signal-Android/search?q=stories&type=commits)?

**Reply by AsamK** [reply]
Not really reverse engineered, just using existing code. signal-cli uses the `service` module from Signal-Android (extracted as libsignal-service).
That code already has the required data structures, that signal-cli just needs to use.
The sending part will be a bit more work, which I'll only do after stories are officially released.
